csharp/carter/web.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>web</AssemblyName>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Carter" Version="[5.1,5.2)" />
```

Comment:
> [5.1,5.2) will get you 5.1.{lowest}. You need to say 5.1.* to get the latest patch

This was something I just learnt. Do you happen to have any reference handy? It's the first time since Go Modules that I've seen a package manager that supports a "minimum matching version the range" logic instead of "maximum version matching the range".

Reply:
me too @rarkins, but I am made the test (with on `F#` framework `here`). we include some `F#` / `C+` for tests, and many languages so we can learn, and its also useful for `renovatebot`